pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -1931,29 +1920,40 @@ func (dr *DashboardServiceImpl) deleteDashboardThroughK8s(ctx context.Context, c
 }
 
 func (dr *DashboardServiceImpl) listDashboardsThroughK8s(ctx context.Context, orgID int64) ([]*dashboards.Dashboard, error) {
-	out, err := dr.k8sclient.List(ctx, orgID, v1.ListOptions{})
-	if err != nil {
-		return nil, err
-	} else if out == nil {
-		return nil, dashboards.ErrDashboardNotFound
-	}
+	dashes := make([]*dashboards.Dashboard, 0)
 
-	// get users ahead of time to do just one db call, rather than 2 per item in the list
-	users, err := dr.getUsersForList(ctx, out.Items, orgID)
-	if err != nil {
-		return nil, err
-	}
+	for continueToken := ""; true; {
```

Comment:
mostly just curious because i haven't seen this pattern before. is there a difference in behavior between what you have and defining continue above the `for` block?

```go
continue := ""
for {
```

Reply:
No, just saving a LOC 😄 Thanks for the review!